src/Microsoft.IdentityModel.Clients.ActiveDirectory/IAdalLogCallback.cs

```diff
@@ -76,18 +89,28 @@ public sealed class LoggerCallbackHandler
     {
         private static readonly object LockObj = new object();
 
-        private static IAdalLogCallback _localCallback;
+        /// <summary>
+        /// Flag to enable/disable logging of PII data. PII logs are never written to default outputs like Console, Logcat or NSLog.
+        /// Default is set to false.
+        /// </summary>
+        public static bool PiiLoggingEnabled { get; set; } = false;
```

Comment:
This is an addition to the public API (and therefore we need to increase the release minor number) => 3.18 and not 3.17.4

Reply:
done.